seed/views/main.py

```diff
@@ -244,19 +245,20 @@ def export_buildings(request):
         selected_fields,
     )
 
-    return {
+    return JsonResponse({
```

Comment:
I'm not sure this should work.  This particular line is fine, but if you are calling this as a DRF method, you shouldn't be accessing `request.body` up on line 172 anymore.  You should access `request.data`, which is already a `dict`, so you don't have to `json.loads` it.  I just wouldn't be surprised if this function fails.